ui/webui/README.rst

```diff
@@ -8,9 +8,44 @@ Make sure you have `npm` available (usually from your distribution package).
 These commands check out the source and build it into the `dist/` directory::
 
     git clone https://github.com/rhinstaller/anaconda.git
+    cd anaconda
+    ./autogen.sh && ./configure
     cd ui/webui
     make
 
+Note that this builds only the WebUI code. In most cases, the rest of Anaconda is needed too.
+
+Running the UI interactively
+----------------------------
+
+First, build RPM packages in the CI container, from root of the repository::
+
+    make -f ./Makefile.am anaconda-ci-build
+    make -f ./Makefile.am container-shell
+    ./autogen.sh && ./configure
+    make rpms
+
+Alternatively, you can install all the dependencies and build without using the container::
+
+    sudo ./scripts/testing/install_dependencies.sh -y
+    ./autogen.sh && ./configure
+    make rpms
+
+Note that if you went through this process already, you also need to remove the whole `ui/webui/bots/` directory before calling `make rpms`. Another pitfall is that the `bots` directory is a git repository and will not be removed with `git clean`.
+
+Then, prepare the `~/.ssh/config` file according to instructions in `<test/README.rst>`_
+
+Finally, set up the VM and start it::
+
+    cd ui/webui
+    make ../../updates.img
+    mkdir -p test/images
```

Comment:
This is not needed, (it was needed last week indeed :)) Now there are saved actually our of the repo, see more https://github.com/cockpit-project/bots/#image-location for information about where images are stored.